pkg/clientset/versioned/typed/bundle/v1alpha1/component.go

```diff
@@ -73,23 +75,33 @@ func (c *components) Get(name string, options v1.GetOptions) (result *v1alpha1.C
 
 // List takes label and field selectors, and returns the list of Components that match those selectors.
 func (c *components) List(opts v1.ListOptions) (result *v1alpha1.ComponentList, err error) {
+	var timeout time.Duration
+	if opts.TimeoutSeconds != nil {
+		timeout = time.Duration(*opts.TimeoutSeconds) * time.Second
+	}
```

Comment:
nit: put this in a function

```go
timeout := timeoutFromOpts(opts)

func timeoutFromOpts(opts v1.ListOptions) out time.Duration {
  if opts.TimeoutSeconds != nil {
    out = time.Duration(*opts.TimeoutSeconds) * time.Second
  }
}
```

Reply:
This is autogenerated from the bash script